src/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
+include_directories(${ROOT_DIR}/src)
```

Comment:
To make the references to "minipal/utils.h" work in inc and the pal. My goal is to keep the inc/pal files as close to the runtime ones as possible.

Reply:
```
inc\sigparser.h:#include <minipal/utils.h>
inc\utilcode.h:#include<minipal/utils.h>
inc\utilcode.h:#include <minipal/utils.h>
pal\src\include\pal\sharedmemory.h:#include <minipal/utils.h>
palrt\common.h:#include <minipal/utils.h>
```

Reply:
Why not add the minipal directory directly? Why src/

Reply:
Because the #include <minipal/utils.h> won't work then.  

This will change after I reorganize things under the "shared" directory to "include_directories(${ROOT_DIR}/src/shared)"